src/components/LayerUI.tsx

```diff
@@ -214,15 +228,10 @@ const LayerUI = ({
   };
   const renderCanvasActions = () => (
     <div style={{ position: "relative" }}>
-      <WelcomeScreenDecor
-        shouldRender={renderWelcomeScreen && !appState.isLoading}
-      >
-        <div className="virgil WelcomeScreen-decor WelcomeScreen-decor--menu-pointer">
-          {WelcomeScreenMenuArrow}
-          <div>{t("welcomeScreen.menuHints")}</div>
-        </div>
-      </WelcomeScreenDecor>
-      {renderMenu()}
+      {WelcomeScreenComponents.MenuHint}
```

Comment:
can this not go into `WelcomeScreen` as a `child` since its absolutely positioned?

Reply:
would be hard to positioning. Also note that relative positioning is great for host apps such as Plus, where we offset the menu button and we get the hint positioning for free. So I'm for keeping as is.